data-structures/HeaderDoublyLinkedList.cpp

```diff
@@ -0,0 +1,211 @@
+#include <iostream>
+
+using namespace std;
+
+struct HDNode {
```

Comment:
Please use this naming conventions https://google.github.io/styleguide/cppguide.html#Type_Names